components/01-atoms/date-time/date-time.twig

```diff
@@ -0,0 +1,73 @@
+{# Remove white space caused by all of the new lines in this file. #}
```

Comment:
@Gregg-Hendrix 
1. Add the logic to handle the date+time formatting. Currently, this is just set as TODO
2. Add single quotes around the datetime and class attribute names